src/ploigos_step_runner/step_implementers/generate_metadata/git.py

```diff
@@ -12,7 +12,7 @@
 Configuration Key     | Required? | Default | Description
 ----------------------|-----------|---------|-----------
 `repo-root`           | Yes       | `./`    | Directory path to the Git repo to generate \
-                                            metadata from.
+                                            metadata from. Must not be in a detached head state.
```

Comment:
...so that pre-release value can be determined.

Reply:
Updated!